src/gevent/tests/test__fileobject.py

```diff
@@ -144,6 +144,23 @@ def test_seek(self):
         self.assertEqual(native_data, s)
         self.assertEqual(native_data, fileobj_data)
 
+    def test_seek_raises_ioerror(self):
+        # Issue #1323
+        fd = sys.stdout
+        with self.assertRaises(OSError) as ctx:
+            os.lseek(fd.fileno(), 0, 0)
```

Comment:
Thanks for the thorough test case. But it fails on Windows, and it fails on Python 3. There's an example earlier of doing something only when we're not working with FileObjectThread, and I think there's an example of skipping a test if it's Python 3.